src/Resources/public/reference-entity/record/table/view.tsx

```diff
@@ -37,7 +37,23 @@ const recordView = ({ value, onChange, locale }: { value: Value; onChange: (valu
                 <tbody>
                     {valueUpdater.tableDataRows.map((rowData: TableDataRow, index: number) => {
                         return (
-                            <tr className="AknOptionEditor-row" key={`row_${attributeCode}_${index}`}>
+                            <tr
+                                className="AknOptionEditor-row"
+                                key={`row_${attributeCode}_${index}`}
+                                id={`row_${attributeCode}_${index}`}
+                                data-position={index}
+                                draggable={valueUpdater.tableDataRows.length - 1 !== index ? 'true' : 'false'}
```

Comment:
here check the type of `draggable`. if it is `boolean`, then every not empty string there will be converted to `true` eventually

Reply:
draggable is even `Booleanish` as a type which is `type Booleanish = boolean | 'true' | 'false';` but I like your suggestion better.